packages/manager/modules/pci/src/projects/project/instances/instances.constants.js

```diff
@@ -27,6 +26,23 @@ export const POLLER_INSTANCE_NAMESPACE = {
   UNSHELVE: 'cloud.project.instance.unshelve',
 };
 
+export const POLLER_INSTANCES = [
+  {
+    needsPolling: (i) => new Instance(i).isShelving(),
```

Comment:
Question: Wouldn't it be better to instantiate the `Instance` object on the caller side and give it as a parameter, to prevent from duplicating instances ?

Reply:
I need to recheck on this